docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
 
 # Failsafe Tab
 
+Failsafe is the action your craft takes when it loses connection to the transmitter.
+This tab allows you to configure what your craft does in an event of a failsafe.
+
+:::danger
+It is very important to set up your failsafe correctly to prevent your craft from flying away or crashing when you lose connection.
+:::
+
+![Failsafe tab](/img/betaflight_configurator_failsafe_tab.png)
+
+## Failsafe Stages
+
+There are two stages of failsafe:
+
+- **Stage 1** - entered when a flight channel has an invalid pulse length, the receiver reports failsafe mode or
+  there is no signal from the receiver at all, the channel fallback settings are applied to all channels and
+  a short amount of time is provided to allow for recovery.
```

Comment:
Not sure exactly what is meant here - could say
Otherwise, Stage 1 duration is configured using the `failsafe_delay` duration, and defaults to 1.5s.